gensim/models/fasttext.py

```diff
@@ -277,21 +277,15 @@
 
 import logging
 import os
+from collections.abc import Iterable
```

Comment:
Curious: is there a tool which reorders/regroups imports per your preferences?

Reply:
`black`, probably. But my preferences = standard PEP8: blocks of "built-in" - "3rd party" - "local" imports (in that order).

Reply:
Plus my editor (sublime + pylint) highlights mixed up imports, even though I still have to reorder them manually (if I choose to).